integration-tests/src/test/java/uk/gov/di/authentication/api/IPVCallbackHandlerIntegrationTest.java

```diff
@@ -301,6 +305,98 @@ void shouldRedirectToLoginWhenSuccessfullyProcessedIpvResponse() throws Json.Jso
                 startsWith(TEST_CONFIGURATION_SERVICE.getLoginURI().toString()));
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
```

Comment:
Not suggesting any changes as I think this approach is quite neat but I think there's a case for writing two separate tests (with more descriptive names) rather than passing `validLoC` here.